src/main/java/com/softserveinc/Task03.java

```diff
@@ -24,5 +25,27 @@ public static void main(String[] args) {
         Scanner scanner = new Scanner(System.in);
         // TODO: Write your code here
 
+        int n;
+        int[] arr = new int[0];
+        do {
+            System.out.print(">>> ");
+            n = scanner.nextInt();
+            if (n <= 0)
+                break;
+            arr = Arrays.copyOf(arr, arr.length + 1);
```

Comment:
You don't need the array to solve this task

Reply:
Thanks! I have refactored the code to use Math.min() instead of array in this case.